feg/gateway/services/eap/providers/aka/servicers/handlers/auth_utils.go

```diff
@@ -93,6 +95,8 @@ func createChallengeRequest(
 		if se, ok := err.(interface{ GRPCStatus() *status.Status }); ok {
 			errCode = se.GRPCStatus().Code()
 		}
+		errMsg := fmt.Sprint("IMSI:", string(lockedCtx.Imsi))
+		err = errors.Wrap(err, errMsg)
```

Comment:
Is it possible to use `fmt.Errorf` to save importing a whole library just for that??

```
fmt.Errorf( "IMSI %s: %s", string(lockedCtx.Imsi), err)
```

In go it is fine to nest errors separated using `:` In fact that can help you to trace back the origin of the error.

Reply:
Resolved the comment by using suggested changes.